src/components/app/data/services/contentHighlights.js

```diff
@@ -22,11 +23,8 @@ export async function fetchEnterpriseCuration(enterpriseUUID, options = {}) {
     // Return first result, given that there should only be one result, if any.
     return data.results[0] ?? null;
   } catch (error) {
-    const errorResponseStatusCode = getErrorResponseStatusCode(error);
-    if (errorResponseStatusCode === 404) {
```

Comment:
[inform] API does not return a 404 when there are no results; no need to check 404 status code explicitly.